infrastructure/persistence/database/reader_simulation_repository.py

```
@@ -0,0 +1,120 @@
+               ) latest ON rs.novel_id = latest.novel_id
+                       AND rs.chapter_number = latest.chapter_number
+                       AND rs.created_at = latest.max_created
+               ORDER BY rs.chapter_number""",
+            (novel_id,),
+        )
+        return [dict(row) for row in cursor.fetchall()]
+
+    def get_high_churn_chapters(
+        self, novel_id: str, threshold: float = 60.0,
+    ) -> List[Dict]:
+        """获取劝退风险高的章节（用于告警）。"""
```

Ruff (0.15.11)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)